internal/action/delete.go

```diff
@@ -63,7 +63,7 @@ func (s *Action) deleteKeyFromYAML(ctx context.Context, name, key string) error
 		return ExitError(ExitIO, err, "Can not delete key '%s' from '%s': %s", key, name, err)
 	}
 	sec.Del(key)
-	if err := s.Store.Set(ctxutil.WithCommitMessage(ctx, "Updated Key in YAML"), name, sec); err != nil {
+	if err := s.Store.Set(ctxutil.WithCommitMessage(ctx, "Updated Key"), name, sec); err != nil {
```

Comment:
Wouldn't it be best to have the name of the entry in the commit message as well?

Reply:
the name of the secret or the name of the key?
adding the key would leak content of the secret to the plain commit history. not a good idea.
the name of the secret is visible in the commit anyway. if there is any use we can duplicate that information to the commit message, but I don't see why.